tests/commands/help.spec.ts

```diff
@@ -27,7 +27,7 @@ Primary commands:
 
     help [command...]                       show help
     login                                   login to balena
-    push <applicationOrDevice>              Start a remote build on the balena cloud build servers or a local mode device
+    push <applicationordevice>              start a remote build on the balena cloud build servers or a local mode device
```

Comment:
It'd be nice to be able to either keep the capitalization or switch to `application_or_device` to make it more readable, with no differentiation between the words it loses a lot of clarity

Reply:
Yeah I agree - this is an issue with the way the capitano based help command displays oclif commands.  This will be fixed when we move the help command to oclif (last command to be converted, about 4 away now).